docs/scripts.js

```diff
@@ -25,9 +26,9 @@ document.addEventListener('DOMContentLoaded', function() {
         document.documentElement.style.setProperty('--primary-color', color);
 
         // Show or hide portfolio items
-        portfolioItems.forEach(item => {
```

Comment:
`.children` returns a `HTMLCollection` which `forEach` doesn't work on. Could either convert it an array, or use a for loop instead (as I've done here).